modules/ztp-recommended-cluster-mc-crs.adoc

```diff
@@ -4,30 +4,44 @@
 
 :_module-type: REFERENCE
 [id="ztp-recommended-cluster-mc-crs_{context}"]
-= Recommended cluster MachineConfig CRs
+= Recommended cluster MachineConfig CRs for {sno} clusters
 
 Check that the `MachineConfig` custom resources (CRs) that you extract from the `ztp-site-generate` container are applied in the cluster. The CRs can be found in the extracted `out/source-crs/extra-manifest/` folder.
 
 The following `MachineConfig` CRs from the `ztp-site-generate` container configure the cluster host:
 
-.Recommended MachineConfig CRs
+.Recommended {ztp} MachineConfig CRs
 [cols=2*, options="header"]
 |====
-|CR filename
+|MachineConfig CR
 |Description
 
-|`02-workload-partitioning.yaml`
-|Configures workload partitioning for the cluster. Apply this `MachineConfig` CR when you install the cluster.
```

Comment:
Should this be left in place since the PerformanceProfile controlled version is not yet GA?

Reply:
Yes. I've added a clarification. Let me know what you think.